src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -2273,6 +2273,9 @@ impl Step for RustDev {
         tarball.permit_symlinks(true);
 
         builder.ensure(crate::core::build_steps::llvm::Llvm { target });
+        if target.contains("linux") && target.contains("x86_64") {
```

Comment:
Oh this was kept in, we forgot to remove it. Please revert the changes to `dist` to make this PR only affect local builds.